src/components/InputBook.js

```diff
@@ -1,6 +1,17 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { booksActions } from '../redux/books/books';
 
 function InputBook() {
+  const dispatch = useDispatch();
+  const handleAddBook = () => {
+    dispatch(booksActions.addBook({
+      id: 1,
+      title: 'Awa and Other plays',
+      author: 'Ndorrh Oswald',
+
+    }));
```

Comment:
- [ ] **Required:** As stated by the previous reviewer, I have added a part of the requirement, you need to implement it such that a new state is returned when the `addBook` action is fired.

![image](https://user-images.githubusercontent.com/27709832/200614971-54a4b431-4e8c-4b6b-9e6f-233f2b3d09a5.png)

Currently, what is added in the form isn't return but the hardcoded data, Kindly fix this.

Reply:
@brytebee  thanks for the review but I think this is not part of yesterdays project.
Please check the requirements of the project of yesterday.
I summited yesterday's projet and not today. 
Please kindly make the review base on the project requirements of yesterday